pkg/apiserver/certificate/common.go

```diff
@@ -0,0 +1,95 @@
+// Copyright 2021 Antrea Authors
```

Comment:
maybe name it "config.go", otherwise many code would be added to this file by having such a meaningless file name.

Reply:
done